app/controllers/ApiController.scala

```diff
@@ -288,8 +287,14 @@ final class ApiController @Inject()(
                   case (newVersion, channel, tags) =>
                     val update =
                       if (formData.recommended)
-                        service.update(project.copy(recommendedVersionId = Some(newVersion.id.value)))
-                      else Future.unit
+                        service.update(
+                          project.copy(
+                            recommendedVersionId = Some(newVersion.id.value),
+                            lastUpdated = new Timestamp(new Date().getTime)
```

Comment:
Can you change the copyWith method to take the right time instead. This won't have any effect